<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eol_sub_window</class>
 <widget class="QWidget" name="eol_sub_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="pin_check_groupBox">
     <property name="title">
      <string>引脚状态检测</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="pin7_label">
          <property name="text">
           <string>pin7状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pin7_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pin8_label">
          <property name="text">
           <string>pin8状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pin8_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="sn_groupBox">
     <property name="title">
      <string>SN读写 二维码</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>SN Number：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sn_number_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="sn_write_label">
          <property name="text">
           <string>写入状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sn_write_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sn_read_label">
          <property name="text">
           <string>读取状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sn_read_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="device_info_groupBox">
     <property name="title">
      <string>设备信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="soft_ver_label">
          <property name="text">
           <string>软件版本：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="soft_ver_lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hw_ver_label">
          <property name="text">
           <string>硬件版本：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hw_ver_lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="calibration_ver_label">
          <property name="text">
           <string>校准版本：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="calibration_ver_lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="usd_hw_ver_label">
          <property name="text">
           <string>USD硬件版本号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usd_hw_ver_lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="usd_soft_ver_label">
          <property name="text">
           <string>USD软件版本号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usd_soft_ver_lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="usd_boot_ver_label">
          <property name="text">
           <string>USD Boot版本：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usd_boot_ver_lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="ver_test_label">
          <property name="text">
           <string>测试结果：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ver_test_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="vcan_groupBox">
     <property name="title">
      <string>VCAN测试</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="vcan_test_label">
          <property name="text">
           <string>测试结果：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="vcan_test_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>688</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="private_sn_groupBox">
     <property name="title">
      <string>私有SN</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="chip_sn_label">
          <property name="text">
           <string>芯片SN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="chip_sn_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="pmic_sn_label">
          <property name="text">
           <string>PMIC SN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pmic_sn_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="client_info_groupBox">
     <property name="title">
      <string>客户信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="did_hw_sf_ver_label">
          <property name="text">
           <string>DID硬件软件版本</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="did_hw_sf_ver_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="voltage_groupBox">
     <property name="title">
      <string>电压测试</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="voltage_type_label">
          <property name="text">
           <string>电压类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="voltage_type_comboBox">
          <item>
           <property name="text">
            <string>NULL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="voltage_label">
          <property name="text">
           <string>电压数值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="voltage_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QComboBox" name="wdg_opt_comboBox">
       <item>
        <property name="text">
         <string>开启看门狗喂狗</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关闭看门狗喂狗</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="write_pushButton">
       <property name="text">
        <string>写入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="test_pushButton">
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="wdg_opt_status_label">
     <property name="text">
      <string>NULL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
